lib/migration.js

```diff
@@ -3,60 +3,69 @@
 // This file is licensed under the Artistic License 2.0.
 // License text available at https://opensource.org/licenses/Artistic-2.0
 
-'use strict';
-const SG = require('strong-globalize');
+"use strict";
+const SG = require("strong-globalize");
```

Comment:
Please fix your editor/IDE settings to use single quotes `'` and revert all changes in quoting style.